proposals/proposal_1/lib/src/app.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:proposal_1/src/items/item_controller.dart';
+import 'package:proposal_1/src/items/item_details.dart';
+import 'package:proposal_1/src/items/item_list.dart';
+import 'package:proposal_1/src/settings/app_theme.dart';
+import 'package:proposal_1/src/settings/settings_controller.dart';
+import 'package:proposal_1/src/settings/settings_view.dart';
+
+class MyApp extends StatelessWidget {
+  final SettingsController settingsController;
+  final ItemController itemController;
+
+  const MyApp({
+    Key key,
+    @required this.itemController,
+    @required this.settingsController,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // Listen to the SettingsController for changes to the App Theme. Rebuild
+    // the App whenever the theme changes
```

Comment:
Yah sorry -- these comments are really rough in general! Wanted to give a general idea without documenting everything since I knew we'd change up a lot of this code.

Reply:
I've tried to add high quality comments throughout the codebase. I think I've fixed formatting and adhered to proper sentence structure, but for some reason my grammar in comments is maddeningly inconsistent!

I appreciate you reading through the comments with an eye for detail :)